spec/factories/members.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :member do
+    name 'Chase'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.